core/src/initializers/codeConfig.ts

```diff
@@ -26,7 +26,10 @@ export class CodeConfig extends Initializer {
 
   async start() {
     const configDir = getConfigDir();
-    await CLS.wrap(async () => loadConfigDirectory(configDir));
+    await CLS.wrap(async () => {
+      const { errors } = await loadConfigDirectory(configDir);
+      if (errors.length > 0) throw new Error("code config error");
```

Comment:
Can you throw the errors themselves so they can see what the problem is?

Reply:
They will already be logged!

Reply:
Example output added to description